Resources/Prototypes/_RMC14/Roles/Jobs/Other/Provost/provost_marshal.yml

```
@@ -82,4 +82,4 @@
   id: RMCJobIconProvostMarshal
   icon:
     sprite: _RMC14/Interface/cm_job_icons.rsi
-    state: hudsquad_pvtml
+    state: hudsquad_pvm
```

Asked Whisper and he wants to keep hudsquad_pvtml for marshals
```suggestion
    state: hudsquad_pvtml
```